doc/source/groupby.rst

```diff
@@ -5,15 +5,15 @@
    :suppress:
 
    import numpy as np
-   np.random.seed(123456)
-   np.set_printoptions(precision=4, suppress=True)
    import pandas as pd
-   pd.options.display.max_rows = 15
-   import matplotlib
-   # matplotlib.style.use('default')
+
    import matplotlib.pyplot as plt
```

Comment:
matplotlib is a third party, and pandas is our own library, so if you divide the imports, matplotlib should be with numpy, and pandas separate